boards/FSM/Src/App/states/App_AllStates.c

```diff
@@ -35,38 +35,26 @@ void App_AllStatesRunOnTick100Hz(struct StateMachine *const state_machine)
     static uint8_t           error_count = 0;
 
     // Check for torque plausibility
-    // TODO: JSONCAN
-    // float left_torque_req = (float)App_CanMsgs_dcm_invl_command_message_torque_command_invl_decode(
-    //     App_CanRx_DCM_INVL_COMMAND_MESSAGE_GetSignal_TORQUE_COMMAND_INVL(can_rx));
-
-    // float right_torque_req = (float)App_CanMsgs_dcm_invr_command_message_torque_command_invr_decode(
-    //     App_CanRx_DCM_INVR_COMMAND_MESSAGE_GetSignal_TORQUE_COMMAND_INVR(can_rx));
-
-    float left_torque_req  = 0;
-    float right_torque_req = 0;
-
+    float left_torque_req  = (float)App_CanRx_DCM_LeftInverterCommand_TorqueCommand_Get();
+    float right_torque_req = (float)App_CanRx_DCM_LeftInverterCommand_TorqueCommand_Get();
     float fsm_torque_limit = App_CanTx_FSM_Apps_TorqueLimit_Get();
 
     if (left_torque_req > fsm_torque_limit || right_torque_req > fsm_torque_limit)
+    {
         error_count++;
+    }
     else
-        error_count = 0;
-
-    if (error_count == MAX_TORQUE_PLAUSIBILITY_ERR_CNT)
     {
-        App_CanTx_FSM_Apps_TorquePlausabilityFailed_Set(true);
+        error_count = 0;
     }
 
+    App_CanAlerts_SetAlert(FSM_FAULT_TORQUE_PLAUSABILITY_FAILED, error_count >= MAX_TORQUE_PLAUSIBILITY_ERR_CNT);
+
```

Comment:
Reimplement torque plausability check: If DCM's requested torque > FSM's torque limit, alert will be set. 
(review this)